### ISSUE TITLE
Autoadd scripted authorization of IPs

### ISSUE THREAD

**Issue by aminud** [ab77/netflix-proxy#277]
Based on the readme and above, I try adding IP directly from osx commandline using curl and it is not working.
curl -s http://host:8080/autoadd?ip=ipv4address&username=admin&password=r8k9FjVZpu

The above will also show all the html code of the page.
However the same url insert into chrome browser is working in updating the IPaddress and iptables rules with another issues, the browser will download a file autoadd.gz after each successfully added the IP address.

I am trying to add/del IP address using terminal command instead of using web browser to do this. I figured it was as simple as adding the iptables rules which can easily be done but apparently need to update the db as well. Is there a shell script that can be called to add or remove ip from command line, script that can add/remove IP from iptables and database.

Thank you

**Comment by ab77** [comment]
@kost take a look?

**Comment by aminud** [comment]
I am also wondering if it is possible to remove/stop the web front end since it is not https and instead use bash script to update/add/del for added security.

**Comment by kost** [comment]
Hello!

Have you tried embedding curl commandline in single quotes as & is special in unix/linux commandline, for example:

```
curl -s 'http://host:8080/autoadd?ip=ipv4address&username=admin&password=r8k9FjVZpu'
```

Feature to disable web frontend is not there at the moment. Feel free to implement it. It's single python file ;)

**Comment by aminud** [comment]
Thanks, that worked:) 
Now how do i delete that IP without going to the browser? Is there an autodel that the reverse of the autoadd?

**Comment by aminud** [comment]
test failed on windows, after adding the IP using autoadd, the IP are added to the db and into the iptables. however accessing ip2location still showing ISP public IP.
Adding manually the IP using the browser is still working correctly.
Is there any other places I should check?

Thanks

**Comment by ab77** [comment]
I don't believe so, but you can add one and submit a PR or ask @kost nicely :)
